### DISCUSSION TITLE
How to work on a gradle project in Zed? (Or other way to use Kotlin packages?)

### DISCUSSION THREAD

**Discussion by axoking** [zed-industries/zed#21381] [Feature Requests]
I'm working on a Kotlin project in Zed. I need Gradle for it because I need external packages like `kotlinx` and `ktor`. However Zed isn't able to detect these external packages and marks them as *"Unresolved reference"*, although Gradle is able to detect and run everything. **Is there any way or extension to make Zed detect that Gradle packages, or another way to use external Kotlin packages in Zed?**

Thanks everyone

**Comment by dodalovicgran** [comment]
Would be nice to have such extension

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/community-links
